scss/partials/buttons.scss

```diff
@@ -0,0 +1,8 @@
+@mixin brand-button($padding) {
+  text-decoration: none;
+  padding: $padding 2*$padding;
+  background: $brand-blue;
+  color: #FFF;
+  margin-left: $padding * -2;
+  @include border-radius(2*$padding);
```

Comment:
What do you think about removing the border radius on this thing? Rounded corners is out, right?